pkg/services/serviceaccounts/manager/service.go

```diff
@@ -70,9 +69,6 @@ func ProvideServiceAccountsService(
 
 	usageStats.RegisterMetricsFunc(s.getUsageMetrics)
 
-	serviceaccountsAPI := api.NewServiceAccountsAPI(cfg, s, ac, accesscontrolService, routeRegister, permissionService)
```

Comment:
You can remove `permissionService`, `ac` and `routeRegister` from the arguments of `ProvideServiceAccountsService`.

Reply:
Good catch! Dependencies removed